data_retriever/migration_event_queue.py

```
@@ -0,0 +1,204 @@
+            EventQueueException: If status could not be sent
+        """
+        if not self._conn or not self._cursor:
+            raise EventQueueException(f"Postgres connection not established")
+        try:
+            self._cursor.execute("""
+                INSERT INTO "history_event" ("entity", "entityId", "action", "userAgent", "createdAt")
+                VALUES ('migration', 'migration_status', %s, 'UPSTRA', NOW());
+            """, (status,))
+            self._conn.commit()
+        except Exception as e:
+            raise EventQueueException(f"Failed to push {status} status to Postgres: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)